python/lsst/ip/diffim/getTemplate.py

```diff
@@ -538,9 +516,9 @@ def setDefaults(self):
         self.coaddPsf.warpingKernelName = 'lanczos5'
 
 
-class GetMultiTractCoaddTemplateTask(pipeBase.PipelineTask):
-    ConfigClass = GetMultiTractCoaddTemplateConfig
-    _DefaultName = "getMultiTractCoaddTemplateTask"
+class GetTemplateTask(pipeBase.PipelineTask):
+    ConfigClass = GetTemplateConfig
+    _DefaultName = "getTemplateTask"
```

Comment:
My fault for including the `Task` in the original getTemplate Tasks' `DefaultName`.  Should be `_DefaultName = "getTemplate"`